app/com/linkedin/drelephant/tuning/obt/BaselineManagerOBT.java

```diff
@@ -0,0 +1,32 @@
+package com.linkedin.drelephant.tuning.obt;
+
+import com.linkedin.drelephant.ElephantContext;
+import com.linkedin.drelephant.tuning.AbstractBaselineManager;
+import com.linkedin.drelephant.util.Utils;
+import java.util.List;
+import models.TuningAlgorithm;
+import models.TuningJobDefinition;
+import org.apache.log4j.Logger;
+import org.apache.hadoop.conf.Configuration;
+
+
+public class BaselineManagerOBT extends AbstractBaselineManager {
+  private final Logger logger = Logger.getLogger(getClass());
+
+  public BaselineManagerOBT() {
+    NUM_JOBS_FOR_BASELINE_DEFAULT = 30;
+    Configuration configuration = ElephantContext.instance().getAutoTuningConf();
+    _numJobsForBaseline =
```

Comment:
Are we expecting this configuration to be different for OBT and HBT? If yes, this config should be named differently. If no, it should be read in base class.